mne/time_frequency/tfr.py

```diff
@@ -1289,22 +1303,22 @@ def _plot(self, picks=None, baseline=None, mode='mean', tmin=None,
                 yscale=yscale, mask=mask, mask_style=mask_style,
                 mask_cmap=mask_cmap, mask_alpha=mask_alpha)
 
-            if title is None:
-                if combine is None or len(tfr.info['ch_names']) == 1:
-                    title = tfr.info['ch_names'][0]
+            if title == 'auto':
+                if len(tfr.info['ch_names']) == 1:
```

Comment:
why not leave the `combine is None or ` in the if statement? Now do the same in the else part checking for `if combine is None`.

Reply:
Good point. I'll change that back!